docs/deploy-apisix-on-bare-metal.md

```diff
@@ -48,7 +48,7 @@ Cloud CLI will copy TLS Bundle into installation directory of Apache APISIX to
 avoid permission issue.
 
 > See the
-> [DP Certificate API](https://docs.az-staging.api7.cloud/swagger/#/controlplanes_operation/getCertificates)
+> [DP Certificate API](https://docs.az-staging.api7.cloud/swagger/#/Clusters_operation/getCertificates)
```

Comment:
```suggestion
> [DP Certificate API](https://docs.az-staging.api7.cloud/swagger/#/clusters_operation/getCertificates)
```

Please keep it consistent with the OpenAPI spec when you update the Cloud cluster-related APIs.